examples/sep24/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sep24",
+  "version": "1.0.0",
+  "description": "The sdk can be found [here](https://github.com/stellar/typescript-wallet-sdk).",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node sep24.ts"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@stellar/typescript-wallet-sdk": "^1.1.2",
```

Comment:
Won't it pull this version from npm?

Reply:
ya, if you're thinking of pulling the source code from under the src folder I'm not sure you can do that 🤔  or if it's a good idea for separation of concerns reason

I can look into this tomorrow

Reply:
imo it's fine separate, in it's own sub-module. It's use is essentially for documentation

Reply:
I think it's a good idea just because the code is always up-to-date...

Reply:
Yeah, it should be separate, but I'm curious how easy it is to get dependency from another folder. I was never been able to do it clean, to be honest

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/58/commits/3874415277787e7f71b294edb7f5fe5667aa8b79
looks to be easier than I thought lol. This set up should work. Doesn't affect src build or test, which is what I was worried about.

though note: it won't break the sdk build or anything if out-of-date with the source code. We can add a script or linting-type thing later though to check it